jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/impl/JettyClientUpgradeRequest.java

```diff
@@ -49,14 +49,14 @@ public JettyClientUpgradeRequest(WebSocketCoreClient coreClient, UpgradeRequest
         if (request != null)
         {
             // Copy request details into actual request
-            HttpFields fields = getHeaders();
+            HttpFields.Mutable fields = getHeaders();
             request.getHeaders().forEach(fields::put);
 
             // Copy manually created Cookies into place
             List<HttpCookie> cookies = request.getCookies();
             if (cookies != null)
             {
-                HttpFields headers = getHeaders();
+                HttpFields.Mutable headers = getHeaders();
```

Comment:
This is the same as the variable `fields` declared above, so remove the duplication?